tools/eubfr-cli/commands/resources/delete.js

```diff
@@ -4,24 +4,23 @@ const { exec } = require('child_process');
 
 const shell = promisify(exec);
 
-// Get config
-const config = require(`../../../../config.json`); // eslint-disable-line import/no-unresolved
-
 // Utilities
+const getConfigurations = require('../../lib/getConfigurations');
 const getServiceLocation = require('../../lib/getServiceLocation');
 
 const deleteResources = async () => {
   const resources = ['resources-raw-storage', 'resources-harmonized-storage'];
+  const { stage, region } = getConfigurations();
 
   const operations = resources.map(async resource => {
     const cwd = path.resolve(`${getServiceLocation(resource)}`);
+
     try {
       console.log(`Deleting ${resource} ...`);
       console.time(resource);
-      await shell(
-        `npx sls remove --stage ${config.stage} --region ${config.region}`,
-        { cwd }
-      );
+      await shell(`npx sls remove --stage ${stage} --region ${region}`, {
```

Comment:
Here, you could use `path.resolve(require.resolve('serverless'), '../../bin/serverless')` to get the path of the binary without executing `npx` or `yarn` or whatsoever ;)